html/browsers/browsing-the-web/navigating-across-documents/javascript-url-query-fragment-components.html

```diff
@@ -1,28 +1,30 @@
 <!doctype html>
-<title> javascript url with query and fragment components </title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/browsing-the-web.html#javascript-protocol">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/webappapis.html#run-a-classic-script">
+<link rel="help" href="https://url.spec.whatwg.org/#percent-decode">
+<link rel="help" href="https://encoding.spec.whatwg.org/#utf-8-decode">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-<script>
-var a = null;
-var b = null;
-var c = null;
-</script>
-
+<div id="log"></div>
+<!-- 9, ("nope" ? "yep" : "what";) is just a JavaScript expression to evaluate -->
```

Comment:
What are these numbers about?

Reply:
Those were for me to keep track of which step in https://html.spec.whatwg.org/multipage/browsing-the-web.html#javascript-protocol was the most relevant to the processing of _urlString_ 

Step 9: 

> Let script be the result of creating a classic script given script source, settings, base URL, and the default classic script fetch options.

I can remove them, or make it clearer?

Reply:
It's probably enough to just use "step 4" instead of "4" .

Reply:
lol, works for me ;) I'll do the update today